alacritty.yml

```diff
@@ -625,6 +625,8 @@
 #   - ToggleBlockSelection
 #   - ToggleSemanticSelection
 #       Toggle semantic selection based on `selection.semantic_escape_chars`.
+#   - CenterAroundCursor
```

Comment:
```suggestion
#   - CenterAroundViCursor
```

Probably also a good idea to explicitly point out that it's the vi cursor in the action name. Just to be safe.

Reply:
I mean, it's already in the section "Vi mode exclusive actions", so I felt like it's redundant.

Reply:
It's not though. Even in vi mode the normal cursor position still exists, so you could center around the write-cursor instead of the vi cursor.

Changing the naming is not about pointing out this is a vi action, but pointing out that you're centering around the vi cursor.